### DISCUSSION TITLE
"No App Access" error after migration

### DISCUSSION THREAD

**Discussion by lukas-schaetzle** [directus/directus#8886] [Archive]
I want to update my deployed directus rc 91 to the the newest version rc 98 but ran into some trouble after running the migrations via `directus bootstrap`. The migrations complete successfully but when I start the App and try to login I get the the following error shown in the UI:

using Firefox: `[UNKNOWN] e.currentUser.role is undefined`
using Chrome: `[UNKNOWN] Cannot read properties of undefined (reading 'id')`

I've played around with different version migrations and the first problem appears when trying to update **from rc 92 to rc 93**. Here I get the following error in the App UI:
```
No App Access
This user isn't allowed to use the admin app
```

Note that I've always tried to use the (only) user to login (with admin rights) and I've tested it with a fresh directus installation.

Any ideas how to fix that?

**Comment by joselcvarela** [comment]
Hello @Tummerhore 
What database are you using?

If you are using SQLite there is this issue https://github.com/directus/directus/issues/8142 we already faced in the past.
The solution for me was to access the database directly and fix the `role` column on `directus_users` table.
You can access it via CLI or using a friendly tool, like https://sqlitebrowser.org/

**Reply by lukas-schaetzle** [reply]
Yes, I've used SQLite for the test indeed. I've tried your query and that fixed it. Many thanks 👍
